regression/cbmc-java/inherited_static_field4/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+test.class
+
+^VERIFICATION (SUCCESSFUL)|(FAILED)$
```

Comment:
This regex seems wrong, accordding to debuggex matches `VERIFICATION SUCCESSFUL` or `FAILED`, I think it should be:

```
^VERIFICATION (SUCCESSFUL|FAILED)$
```